### ISSUE TITLE
Nebraska Helm Chart Repo not found

### ISSUE THREAD

**Issue by tstollin** [flatcar/nebraska#652]
## Description

The Helm chart repo https://kinvolk.github.io/nebraska/ (or https://kinvolk.github.io/nebraska/index.yaml respectively) returns 404.

## Impact

We are no longer able to use the Helm chart of Nebraska.

## Environment and steps to reproduce

1. **Set-up**:  none
2. **Task**: none
3. **Action(s)**:
  a. Open the link https://kinvolk.github.io/nebraska/index.yaml (HTTP GET)
4. **Error**: HTTP 404

## Expected behavior

See the index.yaml of the Helm repo.

**Comment by tafkam** [comment]
same problem here:

```
helm repo add nebraska https://kinvolk.github.io/nebraska/
Error: looks like "https://kinvolk.github.io/nebraska/" is not a valid chart repository or cannot be reached: failed to fetch https://kinvolk.github.io/nebraska/index.yaml : 404 Not Found
```

**Comment by pothos** [comment]
That's a fallout of the repo move to the flatcar org

**Comment by tstollin** [comment]
So, I suppose https://flatcar.github.io/nebraska/index.yaml is the correct url now?

**Comment by pothos** [comment]
Yes, thanks for having a look - there are probably some other things to do as well because the github packages (i.e. container images) don't get forwarded (the git repo does have a forward).

**Comment by tylerauerbeck** [comment]
@ervcz @tormath1 Seems like this issue is fairly old and helm chart references have been updated in the docs and likely have been updated by end users by this point. Probably okay to close?
